src/dswx_sar/dswx_s1.py

```diff
@@ -22,12 +32,30 @@ def dswx_s1_workflow(cfg):
     # Create mosaic burst RTCs
     mosaic_rtc_burst.run(cfg)
 
-    # preprocessing (relocating ancillary data and filtering)
-    pre_processing.run(cfg)
```

Comment:
Any reason that we're not doing the preprocessing? `initial_threshold.run(cfg)` attempts to load filtered image [here](https://github.com/oberonia78/DSWX-SAR/blob/10134e5377c8e5415e079e61fae90e8a995f5572/src/dswx_sar/initial_threshold.py#L1496), and I think we need preprocessing?

Reply:
Yes. You are right. It should be included.